app/controllers/api/databases.php

```diff
@@ -3279,7 +3279,7 @@ function updateAttribute(
             $permissions = $document->getPermissions() ?? [];
         }
 
-        $data = \array_merge($document->getArrayCopy(), $data); // Merge existing data with new data
+        $data = $data + $document->getArrayCopy(); // Merge existing data with new data
```

Comment:
I think @fanatic75 solve this issue with array_merge_recursive?

Reply:
Yeah, `array_merge_recursive` might be worth exploring